ROMFS/px4fmu_common/init.d/13002_firefly6

```diff
@@ -0,0 +1,14 @@
+#!nsh
+#
+# Generic configuration file for BirdsEyeView Aerobotics FireFly6
+#
+# Roman Bapst <romanbapst@yahoo.de>
+#
+
+sh /etc/init.d/rc.vtol_defaults
+
+set MIXER FMU_FireFly6
+
+set PWM_OUT 123456
```

Comment:
Is there a link where this is described, I anyway would like to know more about this?